src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,144 @@
+//SCSS variables
+$gray_color: #777;
+$lightgray_color: #D8D8D8;
+$sky_color: #00BFFF;
+$font_nova: 'Nova Flat';
+$gray_border: 0.19rem solid $gray_color;
+$lightgray_border: 0.0625rem solid $lightgray_color;
+$sky_border: 0.125rem solid $sky_color;
+
+.dashboard-screen {
+    display: flex;
+    margin: 0rem 1.9rem;
+    border-right: $gray_border;
+    font-family: $font_nova;
+}
+
+.dash-left {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    margin-right: 1.9rem;
+}
+
+.user-data {
+    flex: 1;
+    padding: 1.25rem 2.5rem;
+    border: $gray_border;
+    margin-bottom: 1.9rem;
+}
+
+.history {
+    flex: 3;
+    padding: 1.25rem 2.5rem;
+    border-left: $gray_border;
+    border-bottom: $gray_border;
+}
+
+.dash-right {
+    flex: 2;
+    display: flex;
+    flex-direction: column;
+}
+
+.form-group {
+    flex: 1;
+    padding: 1.25rem 2.5rem 1.25rem 0.625rem;
+    border-top: $gray_border;
+    margin-bottom: 1.9rem;
+}
+
+.weather-desc-container {
+    flex: 3;
+    margin-top: 3rem;
+    padding: 1.25rem 2.5rem 1.25rem 0.625rem;
+    border-bottom: $gray_border;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    // justify-content: center;
+}
+.weather-img > img {
+    width: 3.6rem;
+    height: 3.6rem;
+}
+.your_profile_title {
+    padding-bottom: 0.75rem;
+    border-bottom: $lightgray_border; 
+}
+.history-title {
+    padding-bottom: 0.75rem;
+    border-bottom: $lightgray_border; 
+}
+.history-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+.history-item {
+    flex: 1;
+    border: 0.125rem solid $lightgray_color;
+    background-color: #D9F6FF;
+    flex-basis: 30%;
+    box-sizing: border-box;
+    text-align: center;
+    //vertical-align: center;
+    padding: 0.625rem;
+    margin: 0.625rem 0.625rem;
+    position: relative;
+}
+.history-item > p {
+    margin-top: 15px;
+}
+.history-item:hover {
+    flex: 1;
```

Comment:
remove all properties that you will inherit from base class .history-item since they are redundant here

Reply:
In scss you write this easyer:
instead of: 
```
.history-item {
 }
.history-item > p {
 }
.history-item:hover {
 }
```
```
.history-item {
    ...
    > p {
        ...
    }
    &:hover {
        ...
    }
 }
```